autodqm/compare_hists.py

```diff
@@ -9,6 +9,9 @@
 from autodqm import cfg
 from autodqm.histpair import HistPair
 import plotly
+import time
```

Comment:
Do we want to keep the time additions included in the master branch?

Reply:
I thought that we might use it to see the time for each hist during the DDOS plan but thinking again, it deosn't need to be in the master branch  for that. It can just be on testbed. I'll remove and update when both are removed.

Reply:
time module and the comment containing root has been removed